src/components/AssetFetcher/AssetFetcher.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from "react";
+import { AppContext } from "../AppContext/AppContext";
+
+interface Props {
+  children: React.ReactNode;
+}
+
+export const AssetFetcher: React.FC<Props> = ({
+  children,
+}): React.ReactElement => {
+  const [fetchComplete, setFetchComplete] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [apiData, setApiData] = useState([]);
+
+  useEffect(() => {
+    fetch("https://botw-compendium.herokuapp.com/api/v3/compendium/all")
+      .then((res) => res.json())
+      .then((responseJson: { data }) => {
+        const { data } = responseJson;
+        setApiData(
+          data?.filter(
+            (result) =>
+              result.category === "materials" || result.category === "monsters"
```

Comment:
Ooh, we're doing monsters too?

Reply:
if we only do `materials` there isn't any meat so recipes would be very limited, which is fine just need to decide